src/Router.js

```diff
@@ -387,36 +376,11 @@ class ComponentScreen extends RequestScreen {
 			router = redirectRouter;
 		}
 
-		Router.activeState = this.maybeParseLastLoadedStateAsJson();
-		Router.activeState = this.router.addRoutingData(path, Router.activeState);
+		Router.activeState = this.router.addRoutingData(path, this.maybeParseLastLoadedStateAsJson());
 
 		if (Router.activeRouter) {
-			var activeRouter = Router.activeRouter;
-			activeRouter.isActive_ = false;
-
-			if (activeRouter !== router) {
-				var activeComponent = Router.getActiveComponent();
-				if (router.reuseActiveComponent && (activeComponent instanceof router.component)) {
-					// This call is important, as otherwise the component will be disposed
-					// after `activeRouter` is updated, since the router won't render
-					// anything this time. We want to reuse it in another router though.
-					activeRouter.getRenderer().skipNextChildrenDisposal(activeRouter);
-					delete activeRouter.components.comp;
-					activeRouter.element = null;
-
-					const data = activeComponent.getRenderer().getData(activeComponent);
-					data.owner = router;
-					data.parent = router;
-					router.components.comp = activeComponent;
-					router.element = activeComponent.element;
-				} else if (activeRouter.firstRenderElement === router.firstRenderElement) {
```

Comment:
The part inside this `else if` is not related to `reuseActiveComponent`, it's important for when different routers receive the same element, so that changing routers can reuse that element between them. I think it will break things in portal and maybe in dashboard too if you remove it.